moveit_ros/visualization/robot_state_rviz_plugin/src/robot_state_display.cpp

```diff
@@ -413,7 +409,8 @@ void RobotStateDisplay::onEnable()
 // ******************************************************************************************
 void RobotStateDisplay::onDisable()
 {
-  robot_state_subscriber_.shutdown();
+  // TODO(JafarAbdi): Does this works .?
+  //  robot_state_subscriber_.shutdown();
```

Comment:
I mean resetting the SharePtr

Reply:
Yes, resetting the shared pointer works.